frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -96,6 +96,9 @@
     "TASKCFG_ALL_SECURITY_KERBEROS_PRIMARY": "hdfs",
     "TASKCFG_ALL_SECURITY_KERBEROS_PRIMARY_HTTP": "HTTP",
     "TASKCFG_ALL_SECURITY_KERBEROS_REALM": "{{service.security.kerberos.realm}}",
+    {{#service.security.kerberos.debug}}
```

Comment:
Note that this conditional section falls away if we decide to *always* supply the flag.

Reply:
I think doing it this way is more consistent with how we've done other things (e.g. let presence of the envvar indicate you should do something, rather then using the value of the envvar)